store/src/main/java/org/gephi/graph/impl/GraphStore.java

```diff
@@ -85,7 +85,7 @@ public GraphStore(GraphModelImpl model) {
         nodeStore = new NodeStore(edgeStore, GraphStoreConfiguration.ENABLE_AUTO_LOCKING ? lock : null, viewStore, GraphStoreConfiguration.ENABLE_OBSERVERS ? version : null);
         nodeTable = new TableImpl<Node>(configuration, Node.class, GraphStoreConfiguration.ENABLE_INDEX_NODES);
         edgeTable = new TableImpl<Edge>(configuration, Edge.class, GraphStoreConfiguration.ENABLE_INDEX_EDGES);
-        timeStore = new TimeStore(this, GraphStoreConfiguration.ENABLE_AUTO_LOCKING ? lock : null, GraphStoreConfiguration.ENABLE_INDEX_TIMESTAMP);
```

Comment:
This I think was already fixed by 642a1df0dc80a23cae3eb67d6260fc67d2a62a76

I finally decided to reset the configuration rather than recreate the instance